impl/pkg/storage/db/postgres/postgres.go

```diff
@@ -109,12 +110,12 @@ func (p postgres) ListAllRecords(ctx context.Context) ([]pkarr.Record, error) {
 
 	var records []pkarr.Record
 	for _, row := range rows {
-		records = append(records, pkarr.Record{
-			K:   row.Key,
-			V:   row.Value,
-			Sig: row.Sig,
-			Seq: row.Seq,
-		})
+		record, err := row.Record()
+		if err != nil {
+			logrus.WithError(err).Warn("invalid record stored in database, skipping")
```

Comment:
this is probably fine for now but we'll want to handle this later. may be worth adding a TODO or a note to come back to this

Reply:
how would we handle it? This should basically never happen, I assumed this would be handled by setting off an alert if this message was ever logged. I guess we could drop the record from the db entirely if this happens?

Reply:
sg